packages/helpers/src/hide.js

```diff
@@ -0,0 +1,95 @@
+import tokens from '@gympass/yoga-tokens';
+import { css } from 'styled-components';
+import { not } from './media';
+
+const {
+  breakpoints: { xxs, xs, sm, md, lg, xl, xxl, xxxl },
+} = tokens;
+
+const hide = isNot => ({
```

Comment:
Could we make it dynamic? If we change the breakpoints, we will need to update the code here too, right?

```js
const breakpointsAsKeys = Object.keys(breakpoints);

const generateMedias = isNot => breakpointsAsKeys.reduce((acc, breakpoint, index) => {
  const breakpointValue = breakpoints[breakpointsAsKeys[index]];

  if (index === 0) {
    acc[breakpoint] = css`
      @media ${not(isNot)} (max-width: ${breakpointValue.width}px) {
        display: none !important;
      }
    `;
    acc[`${breakpoint}-start`] = css`
      @media ${not(isNot)} (min-width: ${breakpointValue.width}px) {
        display: none !important;
      }
    `;

    return acc;
  }

  if (index === breakpointsAsKeys.length) {
    acc[breakpoint] = css`
      @media ${not(isNot)} (min-width: ${breakpointValue.width}px) {
        display: none !important;
      }
    `;

    return acc;
  }

  const nextBreakpointValue = breakpoints[breakpointsAsKeys[index + 1]];

  acc[breakpoint] = css`
    @media ${not(isNot)}
      (min-width: ${breakpointValue.width}px) and
      (max-width: ${nextBreakpointValue.width - 1}px)
    {
      display: none !important;
    }
  `;

  acc[`${breakpoint}-start`] = css`
    @media ${not(isNot)} (min-width: ${breakpointValue.width}px) {
      display: none !important;
    }
  `;

  return acc;
}, {});

const normalMedias = generateMedias(false);
const oppositeMedias = generateMedias(true); // We may need a better name :P

const hide = isNot => isNot ? oppositeMedias : normalMedias;
```

I'm not sure if this code works, it's just to give an idea.

Reply:
Done!